src/storage/statistics_cl.c

```diff
@@ -508,3 +518,255 @@ stats_dump (const char *class_name_p, FILE * file_p)
 
   fprintf (file_p, "\n\n");
 }
+
+/*
+ * stats_ndv_dump () - Dumps the NDV about a class
+ *   return:
+ *   classname(in): The name of class to be printed
+ *   fp(in):
+ */
+void
+stats_ndv_dump (const char *class_name_p, FILE * file_p)
+{
+  MOP class_mop;
+  int error = NO_ERROR;
+  DB_ATTRIBUTE *att;
+  int col_cnt = 0;
+  CLASS_ATTR_NDV class_attr_ndv;
+  int i;
+
+  class_mop = sm_find_class (class_name_p);
```

Comment:
If there is no other reason, I think it would be better to exclude the view.

Reply:
Okay. but this dump can be removal later. so I will not add any check logic.